### ISSUE TITLE
Different icon for different platform

### ISSUE THREAD

**Issue by pmahend1** [VSCodium/vscodium#763]
**Describe the bug**

- Windows:  *blue tree with circular white background*
- Mac :   *blue tree with white background*
- Ubuntu: *has old green icon*

**Please confirm that this problem is VSCodium-specific**
- [x] This bug doesn't happen if I use Microsoft's Visual Studio Code. It only happens in VSCodium.

**Please confirm that the issue/resolution isn't already documented**
- [x] I checked [the Docs page](https://github.com/VSCodium/vscodium/blob/master/DOCS.md) and my issue is not mentioned there.

**To Reproduce**
- Install VSCodium on Windows
- Install VSCodium on Mac
- Install VSCodium on Ubuntu
- 
**Expected behavior**

Same icon should be used on all platforms.

**Screenshots**

**Desktop (please complete the following information):**
 - OS:Windows 10, Mac BigSur , Ubuntu 20
 - Architecture : x64
 - Version : 1.58.0

**Additional context**
NA

**Comment by hazarek** [comment]
i don't want to use white background on mac.

**Comment by pmahend1** [comment]
@hazarek 

> 
> 
> i don't want to use white background on mac.

Icon should be consistent , can you submit a PR to replace Windows and Linux icons with the non-white background one?

**Comment by hazarek** [comment]
> @hazarek
> 
> > i don't want to use white background on mac.
> 
> Icon should be consistent , can you submit a PR to replace Windows and Linux icons with the non-white background one?

I can only send as png or icns. I don't have technical knowledge about linux and windows icons.

**Comment by daiyam** [comment]
I agree that the icons need to be the same. I will make the necessary changes.

**Comment by daiyam** [comment]
Let's keep it active.

**Comment by aflip** [comment]
On pop OS, the codium icon on the taskbar is blank. Is this due to the same issue? and no icon is found in the applications menu.

**Comment by explode** [comment]
The logo on popup dialog boxes looks weird - like an SVG path not rendering properly.  Main app logo also shown (for side-by-side comparison).  Kubuntu 22.04, Linux kernel 5.15.0-72, Qt version: 5.15.3, Graphics platform: X11
![badly rendered logo](https://github.com/VSCodium/vscodium/assets/47583746/358826c3-8012-464c-b3f1-bb534473098d)

**Comment by andre-koch** [comment]
I can't figure it out either. on Kali it shows the correct icon but in Ubuntu its incorrect!
